src/components/BadgeImage/BadgeImage.js

```diff
@@ -16,13 +16,13 @@ const BadgeImage = ({ data: badge, size = 96 }: Props) => {
   return (
     <FastImage
       style={{
-              height: size,
-              width: size,
-          }}
+                height: size,
+                width: size,
+            }}
       source={{
-              uri: badgeUri,
-              priority: FastImage.priority.normal,
-          }}
+                uri: badgeUri,
+                priority: FastImage.priority.normal,
+            }}
```

Comment:
too much indent :D imho it should go like this:
```js
style={{
    height: size.
    width: size,
}}
source={{
    uri: badgeUri,
    priority: FastImage.priority.normal,
}}
```

Reply:
weird, webstorm ide reformats it to this with eslint -fix

Reply:
vscode too, I feel like eslint has a bug or something's up with configuration, because it doesn't care about indenting objects passed as props, you can make any indent and it will pass :D